mail_activity_team/models/mail_activity.py

```diff
@@ -19,6 +19,8 @@ def _get_default_team_id(self, user_id=None):
             )
         return self.env["mail.activity.team"].search(domain, limit=1)
 
+    user_id = fields.Many2one(required=False)
```

Comment:
Why removing this?

Reply:
Yes, that's true, it is my fault. I will review where did this happen

Reply:
After checking the code, it was removed on 14 migration, it should never come back.